tests/e2e/aws_e2e.py

```diff
@@ -83,6 +86,27 @@ def test_end_to_end_boot_volume(self):
         self.aws.ResourceApi(EC2_SERVICE).Volume(boot_volume_copy.volume_id))
     self.assertEqual(self.volumes[-1].volume_id, boot_volume_copy.volume_id)
 
+    # Create and start the analysis VM and attach the boot volume
+    self.analysis_vm, _ = aws.StartAnalysisVm(
+        self.analysis_vm_name,
+        self.zone,
+        10,
+        4,
+        attach_volume=boot_volume_copy,
+        device_name='/dev/sdp'
+    )
+
+    # The forensic instance should be live in the analysis AWS account and
+    # the volume should be attached
+    instance = self.aws.ResourceApi(EC2_SERVICE).Instance(
+        self.analysis_vm.instance_id)
+    self.assertEqual(instance.instance_id, self.analysis_vm.instance_id)
+    for volume in instance.volumes.all():
+      if volume.volume_id == boot_volume_copy.volume_id:
+        return
+    self.fail('Error: could not find the volume {0:s} in instance {1:s}'.format(
+        boot_volume_copy.volume_id, self.analysis_vm_name))
```

Comment:
If something in the VM Creation logic fails, then your snapshotting tests will fail, making the issue harder to locate.

I would just add an extra test instead of adding this block to both tests.